libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -487,10 +488,14 @@ - (NSArray*) getDevSupportInfos
     return devInfos;
 }
 
-- (NSString*) usersToString:(NSArray*)userAccounts {
+- (NSString*) userToString:(SFUserAccount*)user {
```

Comment:
Otherwise it crashes when trying to put a nil in the devInfos array (in the case we have not logged in yet).